invoices/invoiceregistry.go

```diff
@@ -697,6 +697,11 @@ func (i *InvoiceRegistry) processKeySend(ctx invoiceUpdateCtx) error {
 		},
 	}
 
+	if i.cfg.KeySendHoldTime != 0 {
+		invoice.HodlInvoice = true
```

Comment:
out of scope for this PR, but could be cool to have predicates/rules to determine classes of keysends that this should apply to

Reply:
The idea is that that predicate/rule can now live externally. An application can subscribe to all invoices and when keysends are added, inspect the amt, custom records, etc and settle or fail the invoice

Reply:
i was thinking something even more basic, that doesn't require running an external process. e.g. a static file akin to nginx rules

Reply:
It wouldn't cut it for all use cases. Like again the inventory check.

Reply:
But yes, static config file could be easy to work with for some types of users